config/locales/en.yml

```diff
@@ -320,6 +320,41 @@ en:
               - "I’m unable to leave my home for another reason"
             custom_select_error: "Select if you are able to leave your home or not"
   results_link:
+    feel_unsafe:
+      title: "Feeling Safe"
+      results:
+        feel_safe:
+          title: "If you or someone you know are not feeling safe where you live"
```

Comment:
It may not have been you but is this the title of that page?

Reply:
No it's the sub-heading for a set of links for the results in that group. It should be linked to an answer or a group of answers (e.g. Yes and "not sure").

Reply:
oh right, there's some context i missed out on, no worries :)